okhttp-clients/src/main/metrics/okhttp-metrics.yml

```diff
@@ -0,0 +1,44 @@
+options:
+  javaPackage: com.palantir.conjure.java.okhttp
+  javaVisibility: packagePrivate
+namespaces:
+  conjure-java-client.qos:
+    docs: Conjure okhttp client quality of service metrics.
+    metrics:
+      request-permit.slow-acquire:
+        type: timer
+        docs: Concurrency limiter request permit slow acquisition time.
+      request-permit.leak-suspected:
+        type: meter
+        docs: Rate of suspected request permit leaks.
+      request-permit.slow-acquire-tagged:
+        type: timer
+        tags: [serviceClass]
+        docs: Concurrency limiter request permit slow acquisition time. Idential to `slow-acquire`, but tagged by service.
```

Comment:
I'd appreciate if someone with more context could improve the documentation for these, I wasn't able to find any existing documentation.

Reply:
Have pushed some extra paras :)